api/src/main/webapp/WEB-INF/appengine-web.xml.template

```diff
@@ -51,7 +51,7 @@
     <exclude path="static/README.md"/>
   </static-files>
 
-  <instance-class>${GAE_INSTANCE_CLASS}</instance-class>
+  <instance-class>F4</instance-class>
```

Comment:
I believe this is identical across all environments.

Reply:
I don't think it is identical. We use f4 in prod and test. Use f1/f2 in other envs